src/Orleans/Runtime/GrainReference.cs

```diff
@@ -272,6 +272,17 @@ protected virtual int InterfaceId
         /// <summary>
         /// Implemented in generated code.
         /// </summary>
+        protected virtual ushort InterfaceVersion
+        {
+            get
+            {
+                throw new InvalidOperationException("Should be overridden by subclass");
+            }
```

Comment:
Sometimes in the code we just new up a GrainReference. Could it conflict with this new feature?